README.md

```diff
@@ -3,48 +3,51 @@
 API plugins, microservices, and commands for accessing NetCDF data/metadata contained in iRODS data objects.
 
 Currently supported version(s) of iRODS:
-   - 4.2.8
+   - 4.3.1
+   - 4.2.12
 
 OS Distributions presently supported:
+* on iRODS 4.3.1
+   - AlmaLinux 8 (RHEL 8)
+   - Centos 7
+   - Debian 11
+   - Debian 12
+   - Rocky Linux 9 (RHEL 9)
+   - Ubuntu 18.04
+   - Ubuntu 20.04
+   - Ubuntu 22.04
+* on iRODS 4.2.12
    - Centos 7
-   - Ubuntu 16.04
    - Ubuntu 18.04
 
 Automated building and packaging is available as before, but is now done through CMake.
 
-## Building the Plugin for iRODS 4.2+
+## Building the NetCDF Plugin
 
-Make sure the development packages for OpenSSL, NetCDF, HDF5, and iRODS itself are installed on the build machine, as well
-as these build externals:
+Dependencies for building and running the plugin are as follows:
 
-   - `irods-externals-cmake3.11.4-0`
-   - `irods-externals-clang6.0-0`
-   - `irods-externals-boost1.67.0-0`
+   - irods-externals-cmake
+   - irods-externals-clang
+   - irods-externals-clang-runtime
+   - irods-externals-boost
 
-Clone this repository:
-```sh
-git clone -b {BRANCH|TAG|COMMIT} http:/github.com/irods/irods_netcdf
-```
-(Currently  `4.2.8.0` is the only plugin version available on iRODS 4.2+, and you must be running the 4.2.8 release of the server.)
+Also make sure the development packages for OpenSSL, NetCDF, and HDF5, as well as iRODS itself, are installed on the build machine.
```

Comment:
Can't this be part of the list above?

We don't have to provide version numbers so it seems ok to include them in the list.

Reply:
The names of those vary pretty widely. I probably should either
   * make a table by OS stating the specific package names furnishing the requirement; or
   * probably better, omit the mention of this point entirely (since the build process will likely alert the user to whatever requirements are lacking).

Thoughts?

Reply:
> The names of those vary pretty widely. I probably should either
> 
> * make a table by OS-class (Ub/RHEL/Debian) stating the specific package names required (minus version#s as you suggest)

No. We can't list specific packages (w/ version info) because that can become incorrect over time. Remember, we don't list specific versions for the externals packages either.

> * probably better, omit the mention of this point entirely (since the build process will likely alert the user to whatever requirements are lacking).

That could work, but it would be weird because we'd be asking users to run the build just for it to fail immediately. If we know what packages are required, we should document them.

Looking at previous versions of the README, I think what you have is fine as is. Let's let it ride.